src/client.js

```diff
@@ -79,11 +78,10 @@ class Client extends EventEmitter {
 
     this.deserializer.on('data', (parsed) => {
       parsed.metadata.name = parsed.data.name
-      parsed.data = parsed.data.params
       parsed.metadata.state = state
+      parsed.data = parsed.data.params
       debug('read packet ' + state + '.' + parsed.metadata.name)
-      const s = JSON.stringify(parsed.data, null, 2)
-      debug(s.length > 10000 ? parsed.data : s)
+      debug(parsed.data)
```

Comment:
Why did you change this ?

Reply:
When running with `DEBUG=*` - formatted JSON output tends to be over-verbose, especially with buffers

```
{
  "type": "Buffer",
  "data": [
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    ...
  ]
}
```